ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -75,12 +74,30 @@ SettingsPageLayout {
     Component {
         id: newCollectiblesView
 
-        CommunityMintTokenPanel {
+        CommunityNewCollectibleView {
             anchors.fill: parent
-            transactionStore: root.transactionStore
-            tokensModel: root.tokensModel
-            onMintCollectible: root.mintCollectible(address, name, symbol, description, supply,
-                                                    infiniteSupply, transferable, selfDestruct, network)
+            store: root.communitiesStore
+            onMintCollectible: {
+                root.communitiesStore.mintCollectible(root.communityId,
+                                                      root.transactionStore.currentAccount.address, /*TODO use address from SendModal*/
+                                                      artworkSource,
+                                                      name,
+                                                      symbol,
+                                                      description,
+                                                      supply,
+                                                      infiniteSupply,
+                                                      transferable,
+                                                      selfDestruct,
+                                                      chainId)
+            }
         }
+
+        //        CommunityMintTokenPanel {
```

Comment:
Why is this commented out while previously it was not?

Reply:
Removed!